src/test/java/uk/gov/companieshouse/items/orders/api/mapper/CertificateItemMapperTest.java

```diff
@@ -148,6 +159,47 @@ void testCertificateItemDtoToEntityMapping() {
         assertThat(item.getTotalItemCost(), is(dto.getTotalItemCost()));
     }
 
+    @Test
+    void testCertificateItemDtoToEntityMappingNoDefaults() {
+        final CertificateItemDTO dto = new CertificateItemDTO();
+        dto.setId(ID);
+        dto.setCompanyName(COMPANY_NAME);
+        dto.setCompanyNumber(COMPANY_NUMBER);
+        dto.setCustomerReference(CUSTOMER_REFERENCE);
+        dto.setDescription(DESCRIPTION);
+        dto.setDescriptionIdentifier(DESCRIPTION_IDENTIFIER);
+        dto.setDescriptionValues(DESCRIPTION_VALUES);
+        dto.setItemCosts(ITEM_COSTS);
+        dto.setKind(KIND);
+        dto.setPostalDelivery(POSTAL_DELIVERY);
+        dto.setItemOptions(ITEM_OPTIONS_NO_DEFAULTS);
+        dto.setPostageCost(POSTAGE_COST);
+        dto.setTotalItemCost(TOTAL_ITEM_COST);
```

Comment:
A lot of this is duplicated in the other test methods - I think it would clean up the code to move the common `CertificateItemDTO` setting into a private method and used throughout the test class.

Reply:
Sounds good. Done